.github/workflows/spack_build.yml

```diff
@@ -146,8 +146,6 @@ jobs:
               all:
                 require: "%gcc"
               # https://spack.readthedocs.io/en/latest/packages_yaml.html#setting-requirements-on-virtual-specs
-              blas:
-                require: "openblas"
```

Comment:
I don't understand why this didn't work. It worked for cmake.

I would consider reporting this as a spack bug, but it's near impossible to debug GitHub actions runners sometimes since they run in a special VM

Reply:
This didn't work either... Trying more extreme measures.